### DISCUSSION TITLE
Splits in column by height

### DISCUSSION THREAD

**Discussion by kadanes** [rxhanson/Rectangle#435] [Ideas]
I was wondering if we can have support for half or one third hight in 1/3rd column? If not a drag action then status bar action would work too. Or maybe you can have option to customize what drag actions do?

Something like this:
<pre>
------------------------
|             | 1/3    |
|      2/3    |  x 1/2 | 
|      X 1    | _____  |
|             | 1/3    |
|             |  x 1/2 |
------------------------
</pre>

**Comment by rxhanson** [comment]
I think what you're looking for is sixths, which you can find in the menu & as keyboard shortcuts. (Sixths here is a 2 rows 3 cols not 1 row 6 cols)

Drag action customization has been asked before. I don't currently have the bandwidth to add it in, but don't have a reason to turn down a good pull request for it.

**Reply by kadanes** [reply]
I wanted 1/3 width column and 1/2 hieght. But if I press  `⌃⌥↑` to go to top half and then press `⌃⌥G` to go to 3rd 1/3rd then it takes the whole 1/3rd column instead of just top half. And `⌃⌥↑G`, the output depends on which key (↑/G) was tapped first.

**Reply by rxhanson** [reply]
Combining keys like `⌃⌥↑G` isn't supported the way that Rectangle handles global keyboard shortcuts, as it would have to listen to keystrokes in order to accomplish this. The simplicity of going without keystroke listening outweighs the need for combined non-modifier key shortcuts in my opinion.

It's possible to have stateful commands, where a top half followed by a right third would result in top right third, but it would add a decent amount of complexity to the app - both from a user and developer perspective.

Unless I don't understand what you mean, you can achieve the desired "1/3 width column and 1/2 height" in Rectangle with a single shortcut in the mentioned "Sixths" category:

<img width="499" alt="Screen Shot 2021-04-11 at 9 33 52 PM" src="https://user-images.githubusercontent.com/13651296/114318541-bf390f80-9b0d-11eb-87c8-2f735dc1f4b1.png">

<img width="869" alt="Screen Shot 2021-04-11 at 9 35 03 PM" src="https://user-images.githubusercontent.com/13651296/114318569-dd9f0b00-9b0d-11eb-9968-20b8e7e138c7.png">

**Reply by kadanes** [reply]
Thanks this helps
